homeassistant/components/automation/__init__.py

```diff
@@ -404,6 +409,12 @@ def started_action():
             await self.action_script.async_run(
                 variables, trigger_context, started_action
             )
+        except (vol.Invalid, TemplateError, Unauthorized, ServiceNotFound) as err:
```

Comment:
How many of these inherit from `HomeAssistantError` ?

Reply:
All except for `vol.Invalid`. Does it make sense to just catch `HomeAssistantError` then?
I'm not to familiar with all the situation in which it might be raise, which is why I haven't used it.

Reply:
(and so we can just catch HomeAssistantError)